docs/source/installation.rst

```diff
@@ -115,7 +115,6 @@ Alternatively, you can use Julia's builtin package manager.
 
 .. code-block:: jlcon
 
-   julia> using Pkg  # for julia ≥ 0.7
```

Comment:
using Pkg
is still needed, or ]add PyCall

Reply:
Indeed. Thanks a lot!